aws-globalaccelerator-endpointgroup/src/main/kotlin/software/amazon/globalaccelerator/endpointgroup/CreateHandler.kt

```diff
@@ -69,6 +71,15 @@ class CreateHandler : BaseHandler<CallbackContext?>() {
                     .build()}
     }
 
+    private fun getPortOverrides(overrides: List<PortOverride>?): List<software.amazon.globalaccelerator.endpointgroup.PortOverride>? {
```

Comment:
Shared code between CreateHandler and ReadHandler? Can we DRY it out?

Reply:
Addressed in latest rev.